target/linux/lantiq/xrx200/base-files/etc/board.d/02_network

```diff
@@ -15,9 +15,17 @@ lantiq_setup_interfaces()
 	arcadyan,vg3503j)
 		ucidef_set_interface_lan "lan1 lan2"
 		;;
-	arcadyan,arv7519rw22)
+	arcadyan,arv7519rw22|\
+	avm,fritz3490|\
+	avm,fritz3490-micron|\
+	avm,fritz7490|\
+	avm,fritz7490-micron)
 		ucidef_set_interface_lan "lan1 lan2 lan3 lan4 lan5"
 		;;
+	avm,fritz5490|\
+	avm,fritz5490-micron)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 lan5" "wan"
```

Comment:
I don't see a lan4 port in dts for this device.

Reply:
I will remove the lan4 as long as the qca8k is not working. I cannot remember if it was the testers of 5490 or my own tests with early gswip, that the switch sometimes behaved weird, if either the port to the other switch or the wasp fixed port were not defined. But I have not seen that for a while.